cmd/ui.go

```diff
@@ -331,11 +332,17 @@ func showProgress(
 			return
 		case <-winch:
 			// More responsive progress bar resizing on platforms with SIGWINCH (*nix)
-			termWidth, _, _ = terminal.GetSize(fd)
+			termWidth, _, err = terminal.GetSize(fd)
+			if err != nil {
+				termWidth = defaultTermWidth
+			}
 		case <-ticker.C:
 			// Default ticker-based progress bar resizing
 			if winch == nil {
```

Comment:
Hmm I think that if we fail to get the terminal width initially, we probably shouldn't even attempt to do it again?

Reply:
Hastily resolved in d17100b0?